tests/data/dummy_service_config_files/service_2.yaml

```diff
@@ -0,0 +1,46 @@
+name: dummy_service
+author: valory
+version: 0.1.0
+description: A dummy service config file with sigular overrides.
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: bafybeiapubcoersqnsnh3acia5hd7otzt7kjxekr6gkbrlumv6tkajl6jm
+fingerprint_ignore_patterns: []
+agent: valory/hello_world:0.1.0:bafybeiaotnukv7oq2sknot73a4zssrrnjezh6nd2fwptrznxtnovy2rusm
+number_of_agents: 4
+---
+public_id: valory/dummy_skill:0.1.0
+type: skill
+0:
+  models:
+    params:
+      args:
+        message: ${str:"Hello, World!"}
+    benchmark_tool:
+      args:
+        log_dir: /logs
```

Comment:
The old format does not make sense at all

```yaml
benchmark_persistence_params:
  args: &id001
    log_dir: /benchmarks
public_id: valory/hello_world_abci:0.1.0
type: skill
models:
  0:
  - params:
      args:
        hello_world_message: ${SERVICE_HELLO_WORLD_HELLO_WORLD_STRING:str:HELLO_WORLD!}
  - benchmark_tool:
      args: *id001
```

So this is the new proposed format for multiple overrides

```yaml
public_id: {public_id}
type: {component_type}
{agent_index}:
    {overrides}
```

Reply:
on a side note: agent-specific overwrite are necessary but carry additional risk. We may consider adding a check that ensures all agent overwrite the same set of items.

Reply:
Imo that's too restrictive.

Reply:
We still require the templates for the overrides to be defined in the agent spec. 
The proposed changes are for the parsing of the service spec & its format